### PR TITLE
Link Changed

### LINKED ISSUE #2: "GO TO THE WEBSITE" is not working
Screenshot (403)

### PR DESCRIPTION
**Pull request by cyberpvn7**
I have added pdf of that book.

### CODE REVIEW

File: data/books.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
         "rating": 5,
         "subtitle": "ES6 & Beyond. It covers all aspects of javaScript deep down.",
         "topic": "JavaScript",
-        "website": "https://github.com/getify/You-Dont-Know-JS/tree/master/es6%20&%20beyond"
+        /* "website": "https://github.com/getify/You-Dont-Know-JS/tree/master/es6%20&%20beyond" */
```
Comment (atapas) [inline data/books.json]: @Pavan-Kamthane Remove the commented line.

**Review by atapas — CHANGES_REQUESTED** [review]
@Pavan-Kamthane Provided a review comment.

Could you please check the other book links are correct?

File: data/books.json (hunk 1 above)
Comment (cyberpvn7) [reply data/books.json]: Deleted

**Comment by cyberpvn7** [comment]
Other books links are Correct.

**Comment by cyberpvn7** [comment]
Are you merging it ?

**Review by atapas — APPROVED** [review]
@Pavan-Kamthane Thanks for correcting the link and fixing the bug. Looks good. Accepting.

**Pull request merged**